minicluster/src/main/java/alluxio/master/LocalAlluxioCluster.java

```diff
@@ -48,22 +48,34 @@
 
   private boolean mIncludeSecondary;
 
+  private boolean mIncludeProxy;
+
   private LocalAlluxioMaster mMaster;
 
   /**
    * Runs a test Alluxio cluster with a single Alluxio worker.
    */
   public LocalAlluxioCluster() {
-    this(1, false);
+    this(1, false, true);
```

Comment:
why true as default?
i will prefer to set false as default since it will not modify the behavior of most tests
and most tests don't touch proxy functionalities

Reply:
change to false